interfaces/ILanguageTag.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <utility>
```

Comment:
For what do we need the <utility> ??? 
Can you also remove the std:: in front of the string. The system (never tried but at least we thought about it :-) ) could by flipping a compile switch change from char -> wchar, the strings will flip from std::string -> std::wstring, but that will not work if you define std::string, so lets just define string and have thunder define the global string type which might be std::string or std::wstring, depedending on the compile setting.

Reply:
Addressed the requested changes.